examples/ola-recorder.cpp

```diff
@@ -68,6 +68,10 @@ DEFINE_uint32(stop, 0,
               "Time (milliseconds) in show file to stop playback at. If "
               "the show file is shorter, the last look will be held until the "
               "stop point.");
+DEFINE_s_string(start_trigger_channel, stc, "",
+                "Will not start recording until specified universe and "
+                "channel (comma separated) is set to 255. Example (for "
+                "universe 0 and channel 4): --start-trigger-channel 0,4 ");
```

Comment:
I think the argument format I suggested in https://github.com/OpenLightingProject/ola/pull/1742#pullrequestreview-741055956 might make more sense, i.e. `--start-trigger-universe 0 --start-trigger-channel 4`. There's not much benefit in combining the two arguments (apart from fewer arguments), but it makes our parsing and validation a lot harder for limited benefit.

I guess the only question is can you envisage wanting to trigger from one of many channels, e.g. 0,4 or 5,42 or 1,3 and it just trigger when the first one passes it's threshold? If so, we could look at some other syntax, say `0-4,5-42,1-3` , but that's the only real time I can think that combining them with a comma might make sense.